### PR TITLE
Modified DBCollection.ensureIndex

### PR DESCRIPTION
**Pull request by sjdaniels**
I modified this method so that it creates multi key indexes with keys in the order of the arguments.fields array.

### CODE REVIEW

**Comment by marcesher** [comment]
This is great! Please create tests for these. We have one at https://github.com/marcesher/cfmongodb/blob/develop/test/DBCollectionTest.cfc#L542, but these should be supplemented. Thanks!

**Comment by sjdaniels** [comment]
Added a few more ensureIndex tests...

File: test/DBCollectionTest.cfc [hunk 1]
```diff
@@ -551,6 +551,64 @@ import cfmongodb.core.*;
 		assertTrue( arrayLen(result) GT 1, "Should be at least 2: 1 for the _id, and one for the index we just added");
 	}
 
+	function ensureIndex_should_create_index(){
+		dbCol.dropIndexes();
+		local.indexesStart = dbCol.getIndexes();
```
Comment (marcesher) [inline test/DBCollectionTest.cfc:556]: Sean, you're one of those "local" prefixers? I think there's a shot for that ;-)

**Comment by marcesher** [comment]
Thanks a lot!

**Pull request merged**